homeassistant/components/http/view.py

```diff
@@ -40,7 +40,8 @@ def context(self, request):
 
         return Context(user_id=user.id)
 
-    def json(self, result, status_code=200, headers=None):
+    @staticmethod
+    def json(result, status_code=200, headers=None):
```

Comment:
I had to expose this and the `json_message` as static methods so that they could be accessed by things that don't implement `HomeAssistantView`. Not sure if that is acceptable or if it would be preferred to move them out of the class (within the same module)?

Reply:
We should not do this. This is conflation of concerns.

Webhooks should work without any `http` component related things.